docs/src/stories/components/Layout/Stack.stories.jsx

```diff
@@ -0,0 +1,464 @@
+      table: {
+        category: 'Properties',
+      },
+    },
+    narrow_gap_custom: {
+      control: {
+        type: 'text'
+      },
+      description: 'Override a custom value for `gap` for narrow viewports',
+      table: {
+        category: 'Narrow viewport properties'
+      },
+    },
+
+    align: {
+      options: ['stretch', 'start', 'center', 'end', 'baseline'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: `Sets the alignment between items in the cross-axis of the specified direction. For example:
+- If \`direction\` is set to \`block\` (stacks vertically), it controls the horizontal alignment (left, center, right).
+- If \`direction\` is set to \`inline\` (stacks horizontally), it controls the vertical alignment (top, center, bottom).
+
+This property behavior is equivalent to the \`align-items\` Flexbox property.`,
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'stretch',
+        }
+      }
+    },
+    narrow_align: {
+      options: ['inherit', 'stretch', 'start', 'center', 'end', 'baseline'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit',
+        }
+      },
+    },
+
+    // Align wrap
+
+    alignWrap: {
+      options: ['start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Sets how stack lines are distributed, if they `wrap` into multiple lines. This has equivalent behavior to the `align-content` Flexbox property.',
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'start',
+        }
+      }
+    },
+
+    narrow_alignWrap: {
+      options: ['inherit', 'start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit',
+        }
+      },
+    },
+
+    // Spread
+
+    spread: {
+      options: ['start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio',
+      },
+      description: 'Sets how items will be distributed in the stacking direction.', 
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'start',
+        },
+      },
+    },
+
+    narrow_spread: {
+      options: ['inherit', 'start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio',
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Wrap
+
+    wrap: {
+      options: ['wrap', 'nowrap'],
```

Comment:
Maybe this could just be a boolean.

Reply:
I thought about this, but `flex-wrap` has a few more properties. We don't use right now, but we could consider them in the future if browsers make them accessible.